app/views/content/docs.md

```diff
@@ -13,19 +13,32 @@
 
 ### Resources
 
-* [Status](/docs/status)
+* [Applications](/docs/applications)
 * [Artists](/docs/artists)
 * [Artworks](/docs/artworks)
+* [Bidders](/docs/bidders)
+* [Bidder Positions](/docs/bidder_positions)
+* [Bids](/docs/bids)
+* [Collection Items](/docs/collection_items)
+* [Collections](/docs/collections)
 * [Editions](/docs/editions)
+* [Fairs](/docs/fairs)
 * [Genes](/docs/genes)
-* [Profiles](/docs/profiles)
+* [Images](/docs/images)
 * [Partners](/docs/partners)
+* [Profiles](/docs/profiles)
+* [Sales](/docs/sales)
+* [Search](/docs/search)
 * [Shows](/docs/shows)
-* [Fairs](/docs/fairs)
-* [Applications](/docs/applications)
-* [Users](/docs/users)
+* [Status](/docs/status)
 * [User Details](/docs/user_details)
-* [Collections](/docs/collections)
-* [Collection Items](/docs/collection_items)
-* [Images](/docs/images)
-* [Search](/docs/search)
+* [Users](/docs/users)
+
+
+[comment]: # ( ### Undocumented )
+[comment]: # ( * [Collector Profiles](#) )
+[comment]: # ( * [Docs](#)               )
+[comment]: # ( * [Partner Contacts](#)   )
+[comment]: # ( * [Sale Artworks](#)      )
```

Comment:
I did research on SO to change these from HTML comments to markdown links that won't render at all, so no 
```
<!--
* [Missing Endpoints](/routes)
--> 
```
in the client's final markup.